esp-hal/src/spi/master.rs

```diff
@@ -100,6 +96,20 @@ pub enum SpiInterrupt {
     /// This interrupt is triggered when an SPI transaction has finished
     /// transmitting and receiving data.
     TransDone,
+
+    /// Triggered at the end of End_SEG_TRANS transfer in GP-SPI2 slave
```

Comment:
I know this is TRM text, but we probably shouldn't just paste in details like GP-SPI2 - or slave mode in the master driver.

Reply:
agree - and since is the master driver I will remove the part about SLV